HSMP.hpp

```diff
@@ -80,13 +80,36 @@ struct MessageRequest : ClientRequest {
 };
 
 struct BroadcastRequest : ClientRequest {
+  short int tam_msg;
+  std::string msg;
+
+  BroadcastRequest() : ClientRequest('b',kBroadcastRequest)
   ~BroadcastRequest() {}
-  void PrintStructure() const {}
+  void PrintStructure() const {
+    std::cout << "BroadcastRequest:\n";
+    std::cout << "\tmensaje tam: "<<this->tam_msg<<"\n";
```

Comment:
Please formate code before submitting.
We don't like `std::cout<<"\tmensaje tam: "<<this->tam_msg<<"\n";`.
We prefer `std::cout << "\tmensaje tam: " << this->tam_msg << "\n";`

Reply:
ok

Reply:
do it and commited